infra/rg-tf/blueprint.yaml

```diff
@@ -45,23 +54,23 @@ node_templates:
     properties:
       resource_config:
         environment_variables:
-          ARM_SUBSCRIPTION_ID: { get_secret: azure_subscription_id }
-          ARM_TENANT_ID: { get_secret: azure_tenant_id }
-          ARM_CLIENT_ID: { get_secret: azure_client_id }
-          ARM_CLIENT_SECRET: { get_secret: azure_client_secret }
+          ARM_SUBSCRIPTION_ID: { get_input: [ cloud_credentials, azure_subscription_id ] }
+          ARM_TENANT_ID: { get_input: [ cloud_credentials, azure_tenant_id ] }
+          ARM_CLIENT_ID: { get_input: [ cloud_credentials, azure_client_id ] }
+          ARM_CLIENT_SECRET: { get_input: [ cloud_credentials, azure_client_secret ] }
         variables:
-          azure_location_name: { get_input: azure_location_name }
-          rg_name: { get_input: name }
-          rg_tags: { get_input: rg_tags }
-          sg_name: { concat: [ { get_input: name } , '-cfy-agents' ] }
-          vnet_name: { concat: [ { get_input: name }, '-vnet' ] }
-          vnet_tags: { get_input: vn_tags }
-          cidr_block: { get_input: cidr_block }
-          subnet_1_name: { concat: [ { get_input: name }, '-subnet-1' ] }
-          subnet_1_cidr: { get_input: subnet_1_cidr }
-          subnet_2_name: { concat: [ { get_input: name }, '-subnet-2' ] }
-          subnet_2_cidr: { get_input: subnet_2_cidr }
-          sa_name: { concat: [ { get_input: name }, 'sa' ] }
+          azure_location_name: { get_input: [ resource_config, azure_location_name ] }
+          rg_name: { get_input: [ resource_config, name ] }
+          rg_tags: { get_input: [ resource_config, rg_tags ] }
+          sg_name: { concat: [ { get_input: [ resource_config, name ] } , '-cfy-agents' ] }
+          vnet_name: { concat: [ { get_input: [ resource_config, name ] }, '-vnet' ] }
+          vnet_tags: { get_input: [ resource_config, vn_tags ] }
+          cidr_block: { get_input: [ resource_config, cidr_block ] }
+          subnet_1_name: { concat: [ { get_input: [ resource_config, name ] }, '-subnet-1' ] }
+          subnet_1_cidr: { get_input: [ resource_config, subnet_1_cidr ] }
+          subnet_2_name: { concat: [ { get_input: [ resource_config, name ] }, '-subnet-2' ] }
+          subnet_2_cidr: { get_input: [ resource_config, subnet_2_cidr ] }
+          sa_name: { concat: [ { get_input: [ resource_config, name ] }, 'sa' ] }
         source:
           location: https://github.com/jakubcierlik/eaas-terraform/archive/main.zip
```

Comment:
@jrzeszutek same here , should we consider moving this to official repos ?

Reply:
@ahmadiesa-abu I don't have permissions for creating new repos in cloudify-community, so I just moved the zip archives to eaas-example. I think it's not a bad idea to have all of the EaaS code in a single repo